packages/web/src/scss/components/FileUploader/index.html

```diff
@@ -99,6 +99,65 @@ <h3 id="attachments-multipleFile" hidden>Attachments</h3>
   </div>
   <!-- FileUploader: end -->
 
+</section>
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Input Multiple with validation</h2>
+
+  <!-- FileUploader: start -->
+  <div id="example_multipleWithAccept" class="FileUploader" data-toggle="fileUploader">
+
+    <!-- List item template: start -->
+    <template data-spirit-snippet="item">
+      <li class="FileUploaderAttachment" data-populate-field="item">
+        <svg width="24" height="24" aria-hidden="true">
+          <use xlink:href="/icons/svg/sprite.svg#file" />
+        </svg>
+        <span class="text-truncate" data-populate-field="name"></span>
+        <button type="button" class="FileUploaderAttachment__remove" data-populate-field="button">
+          <span class="accessibility-hidden">Remove</span>
+          <svg width="24" height="24" aria-hidden="true">
+            <use xlink:href="/icons/svg/sprite.svg#close" />
+          </svg>
+        </button>
+      </li>
+    </template>
+    <!-- List item template: end -->
+
+    <!-- FileUploaderInput: start -->
+    <div class="FileUploaderInput" data-spirit-element="wrapper">
+      <label for="fileUploadMultipleWithAccept" class="FileUploaderInput__label">Label</label>
+      <input type="file" id="fileUploadMultipleWithAccept" name="attachment3" class="FileUploaderInput__input" multiple data-spirit-element="input" accept=".png,image/jpeg" />
+      <div class="FileUploaderInput__dropZone" data-spirit-element="dropZone">
+        <svg width="24" height="24" aria-hidden="true">
+          <use xlink:href="/icons/svg/sprite.svg#upload" />
+        </svg>
+        <label for="fileUploadMultipleWithAccept" class="FileUploaderInput__dropZoneLabel">
+          <span class="FileUploaderInput__link link-primary link-underlined">Upload your file(s)</span>
+          <span class="FileUploaderInput__dragAndDropLabel">or drag and drop here</span>
+        </label>
+        <div class="FileUploaderInput__helperText">Max size of each file is 10 MB</div>
+      </div>
+    </div>
+    <!-- FileUploaderInput: end -->
+
+    <!-- FileUploaderList: start -->
+    <h3 id="attachments-multipleWithAccept" hidden>Attachments</h3>
+    <ul class="FileUploaderList" aria-labelledby="attachments-multipleWithAccept" data-spirit-element="list"></ul>
+    <!-- FileUploaderList: end -->
+
+  </div>
+  <!-- FileUploader: end -->
+
+  <br />
+  ⚠️
+  <br />
+  <small>️Only JPG and PNG images are allowed in this example.</small>
+  <br />
+  <small>
+    <pre><code>accept=".png,image/jpeg"</code></pre>
+  </small>
```

Comment:
Why not use `helperText` instead of all this?

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/d41060b4-9bb6-49a1-9851-e78c86a1f29e)

Reply:
whatever, fixed